config/crd/bases/etcd.improbable.io_etcdclusters.yaml

```diff
@@ -167,6 +169,21 @@ spec:
           type: object
         status:
           description: EtcdClusterStatus defines the observed state of EtcdCluster
+          properties:
+            members:
+              description: Members contains information about each member from the
+                etcd cluster.
+              items:
+                properties:
+                  ID:
+                    type: string
+                  name:
+                    type: string
+                required:
+                - ID
+                - name
```

Comment:
Does this validation only apply to `/status` subresource API calls? I suppose it does.

Reply:
Unsure. I would assume so. @munnerz?